pkg/apis/application/v1alpha1/generated.proto

```diff
@@ -38,7 +38,7 @@ message AWSAuthConfig {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:path=appprojects,shortName=appproj;appprojs
 message AppProject {
-  optional k8s.io.apimachinery.pkg.apis.meta.v1.ObjectMeta metadata = 1;
+  optional .k8s.io.apimachinery.pkg.apis.meta.v1.ObjectMeta metadata = 1;
```

Comment:
Is this normal (the leading `.`) or a bug during the generation ? 🤔

Reply:
Apparently the leading dot means it's a fully resolved reference instead of relative. https://protobuf.com/docs/descriptors#type-references